telco-core/configuration/reference-crs-kube-compare/optional/other/monitoring-config-cm.yaml

```diff
@@ -0,0 +1,28 @@
+# optional
+# count: 1
+---
+apiVersion: v1
```

Comment:
The sosreport that I'm comparing to has:
```
+  config.yaml: "alertmanagerMain:\t\n  nodeSelector:\t\n    node-role.kubernetes.io/infra
+    : \"\"\t\n  tolerations:\t\n  - effect: NoSchedule\t\n    key: node-role.kubernetes.io/infra\t\n
(...)
```
probably because someone used a patch command. I suppose this works for the partner in question. I expect that other customers might have similar results given that this entire field is a large string.

Also, we must account for customers/partners who use node-selectors and tolerations to move their resources to other nodes - so regex matches are required here.

Reply:
This CR and others which embed complex structured data cause issues not only for this comparison, but in other parts of normal k8s management of the cluster (eg multiple controllers managing different parts of this data). We have a story in the backlog to work on better handling in these cases. (ref CNF-13528)